saundracatalina/crate-1
# Feature/edit profile functionality

### What does this PR do? 
>Clearly and concisely describe the feature. 
This PR add functionality to the user info section, a user will be able to edit their info and have it persist on refresh.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.
We had to edit local state, redux store and local storage to keep new information on page refresh. 
We had difficulty matching all the fields necessary to update the backend, due to our spelling error mistake.
Our current error message states that our server and client are not matching up where we are updating both. 

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
code/api/src/modules/user/mutations.js
code/api/src/modules/user/resolvers.js 
code/web/src/modules/user/EditInfo.js 
code/web/src/modules/user/Profile.js
code/web/src/modules/user/api/actions.js
code/web/src/modules/user/api/state.js
code/web/src/setup/store.js

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. run `npm start` on code/web & code/api
2. Start up `localhost:3000`
3. Login in
4. Go the the Profile button & Click
5. Click on the image with the paper and pencil to edit the user info section
6. Edit user info & click save
7. Refresh page

### Steps To Take
>Any questions and possible steps moving forward.
Would need to connect backend with Items kept, product full history and delivery date section

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
#4  #5  #6  #29
